shell/common/extensions/electron_extensions_client.cc

```diff
@@ -130,9 +130,7 @@ const GURL& ElectronExtensionsClient::GetWebstoreUpdateURL() const {
 }
 
 bool ElectronExtensionsClient::IsBlacklistUpdateURL(const GURL& url) const {
-  // TODO(rockot): Maybe we want to do something else here. For now we accept
-  // any URL as a blacklist URL because we don't really care.
-  return true;
+  return false;
```

Comment:
What does this allow?

Reply:
@nornagon Returning `true` basically blocks all URLs to be processed through code in `web_request_api.cc`.
https://source.chromium.org/chromium/chromium/src/+/master:extensions/browser/api/web_request/web_request_permissions.cc;l=329;drc=5ba77dad98cf94506cf3700f9e12fcdc65fadfb6?originalUrl=https:%2F%2Fcs.chromium.org%2F